content/source/docs/enterprise/workspaces/run-basics.html.md

```diff
@@ -26,17 +26,79 @@ When you request a run, TFE locks the run to the current Terraform code (usually
 
 ## Run States
 
-A given Terraform run can be in the following states:
+Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and TFE shows the progress through those stages as run states.
+
+In the list of workspaces on TFE's main page, each workspace shows the state of the run it's currently processing. (Or, if no run is in progress, the state of the most recent completed run.)
+
+### 1. The Pending Stage
+
+_States in this stage:_
+
+- **Pending:** TFE hasn't started action on a run yet. TFE processes each workspace's runs in the order they were queued, and a run remains pending until every run before it has completed.
+
+_Leaving this stage:_
+
+- Proceeds automatically to the plan stage (**Planning** state) when it becomes the first run in the queue.
+- Can skip to completion (**Discarded** state) if a user discards it before it starts.
+
+### 2. The Plan Stage
+
+_States in this stage:_
 
-- **Pending:** A pending run hasn't been planned or applied. It's waiting for other runs to finish.
 - **Planning:** TFE is currently running `terraform plan`.
-- **Planned:** TFE has run `terraform plan`, and is waiting for a user to approve a `terraform apply`.
+- **Planned:** `terraform plan` has finished. TFE sometimes pauses in this state, depending on the workspace and organization settings.
+
+_Leaving this stage:_
+
+- If the `terraform plan` command failed, TFE skips to completion (**Plan Errored** state).
```

Comment:
**Plan Errored** you have a few places. 

It may be better captured as just **Errored**, since Plan kind of refers to two things:
* The stage of the run: *Your terraform run is in the plan stage*
* Sort of refers to the configuration????: *Your terraform plan has errored in the applying stage*

I don't remember exactly what it says in the UI.

Reply:
It says "Plan Errored" in the UI, so I've been matching that in the docs. But I agree that plain old "Errored" seems like a better UI label for that! I'm in favor of changing in both places, but the UI should probably change first.